input/pagecontent/reference.md

```diff
@@ -2716,7 +2731,7 @@ Rule sets can be inserted in the context of a path. The context is specified by
 <pre><code>* &lt;element&gt; insert {RuleSet}<span class="optional">(value1, value2, value3...)</span>
 </code></pre>
 
-Alternately, the context can be given by indenting the insert rule under another rule that provides a path context (see [indented rules](#indented-rules)).
+Alternately, the context can be given by indenting the insert rule under another rule that provides a path context (see [indented rules](#indented-rules)). The path context from the rule which the insert rule is indented below will be applied to all rules being inserted.
```

Comment:
Would it make sense to provide an example? Or do we have one already?

Reply:
Yes, below we do have 
![image](https://user-images.githubusercontent.com/52747882/134984309-29262ba0-bc12-4e19-b3d9-975119c1ddbb.png)
 which should illustrate how this works.

Reply:
::thumbs up: